ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+#### Subject of the issue
+Describe your issue here
+
+#### Your environment
+* version/branch
+* which browser and its version
```

Comment:
I would add "(optional)" since I think the only time it makes sense to give this information is a bug report